autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/ScalingRequest.java

```diff
@@ -142,7 +142,12 @@ private void scaleUp(int scalingAdjustment, int hostGroupNodeCount) {
             instanceGroupAdjustmentJson.setInstanceGroup(hostGroup);
             updateStackJson.setInstanceGroupAdjustment(instanceGroupAdjustmentJson);
 
-            cloudbreakCrnClient.withInternalCrn().autoscaleEndpoint().putStack(stackCrn, cluster.getClusterPertain().getUserId(), updateStackJson);
+            if (cluster.isStopStartScalingEnabled()) {
+                cloudbreakCrnClient.withInternalCrn().autoscaleEndpoint().putStackStartInstances(stackCrn, updateStackJson);
```

Comment:
why do you need to start the instances instead of the scaling?

Reply:
In stop start scaling approach we stop the instances during scale down instead of terminating them, so during scale up we start those instead of adding new instances.